pkgs/by-name/mo/modern-cpp-kafka/package.nix

```diff
@@ -54,7 +54,7 @@ stdenv.mkDerivation rec {
   propagatedBuildInputs = [ rdkafka ];
 
   cmakeFlags = [
-    "-DLIBRDKAFKA_INCLUDE_DIR=${rdkafka.out}/include"
+    "-DLIBRDKAFKA_INCLUDE_DIR=${rdkafka.dev}/include"
```

Comment:
I notice that there is also `LIBRDKAFKA_LIBRARY_DIR`. Would setting it help make it build on Darwin?

BTW, we could use `${lib.getLib rdkafka}/lib` and `${lib.getInclude rdkafka}/include` instead of hard-code the outputs.

Reply:
So [`LIBRDKAFKA_LIBRARY_DIR`](https://github.com/morganstanley/modern-cpp-kafka/blob/122678e881de94721458fd948f38e65366b68689/CMakeLists.txt#L69) only checks for the static library name on macos, which worked so far, since both were built and in the same folder. Meh. I'll make a PR with the patch later, but can somebody tell me what the library is even named on macos?

(modern-cpp-kafka is a header only library, it has no business doing these checks. And things that do actually need to find the library should use pkg-config. I'm tempted to set `sourceRoot = "${src}/include";` or rip out that CMakeLists otherwise. :/)